### PR TITLE
SUBMARINE-337. MXNET example in mini-submarine

### PR DESCRIPTION
**Pull request by lowc1012**
### What is this PR for?
SUBMARINE-337. Add a MXNET example in mini-submarine

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
[SUBMARINE-337](https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-337)

### How should this be tested?
[passed CI](https://travis-ci.org/lowc1012/submarine/builds/657377080)

### Screenshots (if appropriate)
螢幕快照 2020-03-02 下午10 17 11

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: dev-support/mini-submarine/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,11 @@ RUN \
   apt-get update && \
   apt-get -y install vim
 
+# Install libgomp1
```
Comment (pingsutw) [inline dev-support/mini-submarine/Dockerfile]: change to `# Install libgomp1 for MXNET`

File: dev-support/mini-submarine/submarine/image_classification.py [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+metric = CompositeEvalMetric([Accuracy(), TopKAccuracy(5)])
+kv = mx.kv.create(opt.kvstore)
+
+
+def get_cifar10_iterator(batch_size, data_shape, resize=-1, num_parts=1, part_index=0):
+    get_cifar10()
+
+    train = mx.io.ImageRecordIter(
+        path_imgrec = "data/cifar/train.rec",
+        # mean_img    = "data/cifar/mean.bin",
```
Comment (pingsutw) [inline dev-support/mini-submarine/submarine/image_classification.py]: Could we remove this line?

File: dev-support/mini-submarine/submarine/image_classification.py [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+        data_shape  = data_shape,
+        batch_size  = batch_size,
+        rand_crop   = True,
+        rand_mirror = True,
+        num_parts=num_parts,
+        part_index=part_index)
+
+    val = mx.io.ImageRecordIter(
+        path_imgrec = "data/cifar/test.rec",
+        # mean_img    = "data/cifar/mean.bin",
```
Comment (pingsutw) [inline dev-support/mini-submarine/submarine/image_classification.py]: Could we remove this line?

**Comment by pingsutw** [comment]
Thanks @lowc1012 for the contribution
Overall LGTM, just some minor suggestions in comment

**Comment by lowc1012** [comment]
@pingsutw Thanks for your review.

File: dev-support/mini-submarine/submarine/image_classification.py (hunk 3 above)
Comment (lowc1012) [reply dev-support/mini-submarine/submarine/image_classification.py]: done

File: dev-support/mini-submarine/submarine/image_classification.py (hunk 2 above)
Comment (lowc1012) [reply dev-support/mini-submarine/submarine/image_classification.py]: done

File: dev-support/mini-submarine/Dockerfile (hunk 1 above)
Comment (lowc1012) [reply dev-support/mini-submarine/Dockerfile]: done

**Pull request closed without merging**